docs/section5/Rule5-15.md

```diff
@@ -16,16 +16,16 @@
 
 ## Rule Logic:  
 
-- For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
+- For each building segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
 
-  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+  - For each thermal_block in building segment: ```for thermal_block_b in building_segment_b.thermal_blocks:```  
 
-  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
+  - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
 
-  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
+  - For each space in thermal zone: ```for space_b in zone_b.spaces:```  
 
-    - Get surfaces in space: ```for surface_baseline in space_baseline.surfaces:```  
+    - For each surface in space: ```for surface_b in space_b.surfaces:```  
 
-      - Get surface optics for roof: ```if ( surface_baseline.classification == "CEILING" ) AND ( surface_baseline.adjacent_to == "AMBIENT" ): roof_surface_optics_baseline = surface_baseline.surface_optics```  
+      - Get surface optics for roof: ```if ( surface_b.classification == "CEILING" ) AND ( 0<= surface_b.tilt <60 ) AND ( surface_b.adjacent_to == "AMBIENT" ): roof_surface_optics_b = surface_b.surface_optics```  
 
-      **Rule Assertion:** Baseline roof solar reflectance is 0.30: ```roof_surface_optics_baseline.absorptance_solar_exterior == 0.70```  
+        **Rule Assertion:** Baseline roof solar reflectance is 0.30: ```roof_surface_optics_b.absorptance_solar_exterior == 0.70```  
```

Comment:
@mkarpman @JasonGlazer The revisions @xingcx has made makes sense.
However, we have two RMR data elements that should mean the same thing. Should we be checking both (i.e. tilt + surface classification) or only the tilt? i would vote to check ONLY the tilt. This way we can avoid addressing a scenario where tilt indicates a different surface type than the classification.

Reply:
@supriyagoel I'm not sure if the roof and floor will have different tilt (positive vs negative), so kept both surface classification and tilt.

Reply:
@xingcx that's a good point. So we will also have to check for the surface adjacency. Only surfaces with the adjacency specified as 'Ambient' would be flagged as roofs.